### PR TITLE
Add functionality to jump from Issues-tree to project-descriptor

### PR DESCRIPTION
**Pull request by barbelity**

### CODE REVIEW

File: src/main/java/com/jfrog/ide/idea/scan/GradleScanManager.java [hunk 1]
```diff
@@ -99,6 +105,19 @@ protected void refreshDependencies(ExternalProjectRefreshCallback cbk, @Nullable
         ExternalSystemUtil.refreshProject(project, GradleConstants.SYSTEM_ID, getProjectBasePath(project).toString(), cbk, false, ProgressExecutionMode.IN_BACKGROUND_ASYNC);
     }
 
+    @Override
+    protected PsiFile[] getProjectDescriptors() {
+        String buildgradlePath = Paths.get(Utils.getProjectBasePath(project).toString(), "build.gradle").toString();
```
Comment (yahavi) [inline src/main/java/com/jfrog/ide/idea/scan/GradleScanManager.java:110]:
```suggestion
        String buildGradlePath = Paths.get(Utils.getProjectBasePath(project).toString(), "build.gradle").toString();
```

File: src/main/java/com/jfrog/ide/idea/scan/GradleScanManager.java [hunk 2]
```diff
@@ -99,6 +105,19 @@ protected void refreshDependencies(ExternalProjectRefreshCallback cbk, @Nullable
...
+        VirtualFile file = LocalFileSystem.getInstance().findFileByPath(buildgradlePath);
+        PsiFile psiFile = PsiManager.getInstance(project).findFile(file);
```
Comment (yahavi) [inline src/main/java/com/jfrog/ide/idea/scan/GradleScanManager.java:112]: Please add a null check to the file

File: src/main/java/com/jfrog/ide/idea/scan/NpmScanManager.java [hunk 3]
```diff
@@ -47,5 +54,17 @@ protected void buildTree(@Nullable DataNode<ProjectData> externalProject) throws
         setScanResults(npmTreeBuilder.buildTree(getLog()));
     }
 
+    @Override
+    protected PsiFile[] getProjectDescriptors() {
+        String packagjsonPath = Paths.get(Utils.getProjectBasePath(project).toString(), "package.json").toString();
```
Comment (yahavi) [inline src/main/java/com/jfrog/ide/idea/scan/NpmScanManager.java:59]:
```suggestion
        String packageJsonPath = Paths.get(Utils.getProjectBasePath(project).toString(), "package.json").toString();
```

File: src/main/java/com/jfrog/ide/idea/scan/NpmScanManager.java [hunk 4]
```diff
@@ -47,5 +54,17 @@ protected void buildTree(@Nullable DataNode<ProjectData> externalProject) throws
...
+        VirtualFile file = LocalFileSystem.getInstance().findFileByPath(packagjsonPath);
+        PsiFile psiFile = PsiManager.getInstance(mainProject).findFile(file);
```
Comment (yahavi) [inline src/main/java/com/jfrog/ide/idea/scan/NpmScanManager.java:61]: Please add a null check to the file

File: src/main/java/com/jfrog/ide/idea/ui/issues/IssuesTab.java [hunk 5]
```diff
@@ -207,4 +218,65 @@ public void registerListeners() {
...
+            navigationCandidates = navigationService.getNavigation(selectedNode);
+            if (navigationCandidates == null) {
+                return null;
+            }
+        }
+        PsiElement navigationTarget = navigationCandidates.iterator().next();
+        JMenuItem jumpToElement = new JMenuItem(new AbstractAction(POPUP_MENU_HEADLINE) {
+            @Override
+            public void actionPerformed(ActionEvent e) {
+                if (navigationTarget != null && navigationTarget instanceof Navigatable && ((Navigatable) navigationTarget).canNavigate()) {
```
[the remark this answers is not in the document]
Comment (barbelity) [reply src/main/java/com/jfrog/ide/idea/ui/issues/IssuesTab.java:274]: Took your suggestion for the Roman style 👍

File: src/main/java/com/jfrog/ide/idea/ui/issues/IssuesTree.java [hunk 6]
```diff
@@ -104,4 +115,64 @@ private void calculateIssuesCount() {
...
+        if (!e.isPopupTrigger()) {
+            return;
+        }
+
+        // Event is right-click.
+        TreePath selPath = tree.getPathForLocation(e.getX(), e.getY());
+        if (selPath == null) {
+            return;
+        }
+        DependenciesTree selectedNode = (DependenciesTree)selPath.getLastPathComponent();
```
Comment (yahavi) [inline src/main/java/com/jfrog/ide/idea/ui/issues/IssuesTree.java:140]:
```suggestion
        DependenciesTree selectedNode = (DependenciesTree) selPath.getLastPathComponent();
```

**Pull request merged**